src/snowcli/cli/sql.py

```diff
@@ -112,18 +79,9 @@ def execute_sql(
         schema=schema,
     )
 
-    table = Table(show_lines=True, box=box.ASCII, width=120)
-    table.add_column("Query")
-
-    if verbose:
-        with Live(table, auto_refresh=False) as live:
-            conn.ctx.execute_string(
-                sql_text=sql,
-                remove_comments=True,
-                cursor_class=partial(LoggingCursor, LiveOutput(table, live)),
-            )
-    else:
-        conn.ctx.execute_string(
-            sql_text=sql,
-            remove_comments=True,
-        )
+    results = conn.ctx.execute_string(
+        sql_text=sql,
+        remove_comments=True,
+    )
+    for result in results:
+        print_db_cursor(result)
```

Comment:
Shouldn't it be `print_db_cursor(results)`?

Reply:
No, `print_db_cursor` expects `SnowflakeCursor`, but in this case it gets list of SnowflakeCursors

Reply:
How it works with json format?

Reply:
It treats each result separately 
```
% snow --format JSON sql -q "select 1; select pi(); select current_user"
[
  {
    "1": 1
  }
]
[
  {
    "PI()": 3.141592654
  }
]
[
  {
    "CURRENT_USER": "ADMIN"
  }
]
```

Reply:
Can you pipe it to `jq`?

Reply:
no